app/views/layouts/_links.html.erb

```diff
@@ -10,20 +10,51 @@
     </span>
   </a>
   <div class="card-content" data-card-target="content" hidden>
-    <div class="content">
-      <ul>
-        <li><a href="https://docs.google.com/document/d/15kSw0O1jtZYTeLh3jZg2XeGFbNbavIBsA0u_PvK8YRs/mobilebasic"  target="_blank">
-          Pomoć obiteljima iz Petrinje, Siska i okolice - kontakti, smještaj i prijevoz - Google doc (preview)
-        </a></li>
-        <li><a href="https://docs.google.com/document/d/1yUxHtKCkkXEQ8tXTD2bODbisxwF5Q9zvPJoU27vQGUM/edit?fbclid=IwAR0g1F__7avcVpOlf__NQgBq626LDmDNyz42k545TT5MCjTLMvf4De9tvMs" target="_blank">
-          Pomoć za Petrinju - Google doc
-        </a></li>
-        <li><a href="https://www.facebook.com/groups/155412969344431/permalink/155812429304485/" target="_blank">
-          POMOC GLINI, PETRINJI, SISKU i okolnim mjestima - Facebook grupa
-        </a></li>
-        <li><a href="https://potres2020.openit.hr/views/map" target="_blank">
-          Potres 2020 - mapa pomoći
-        </a></li>
+    <div class="help-category">
+      <div class="media">
+        <div class="media-left">
+          <span class="icon is-large">
+            <%= inline_svg_tag('globe.svg', aria: true) %>
```

Comment:
Not sure if we need yet another icon for this.

Reply:
Which already available icon would you use?

Reply:
No icon at all?

Reply:
To me it looks a bit bland without the icon

![image](https://user-images.githubusercontent.com/1616329/103761427-f1226d80-5016-11eb-9bb9-3f37b8630031.png)